source/triggers/database-triggers.txt

```diff
@@ -234,6 +234,21 @@ options are available:
 
      - .. include:: /includes/trigger-auto-resume.rst
 
+   * - :guilabel:`Maximum Throughput Triggers`
+
+     - .. _triggers-maximum-throughput:
+      
+       If the linked data source is a dedicated server (M10+ Tier), 
+       you can increase the :ref:`maximum throughput <event_processing_throughput>`
+       from the default 10,000 concurrent processes to 50,000. 
+
+       Before increasing the maximum throughput, consider whether one or more of 
+       your triggers are calling a rate-limited external API. Increasing the 
```

Comment:
```suggestion
       Before increasing the maximum throughput, consider whether the
       trigger is calling a rate-limited external API. Increasing the 
```

The limit & setting are per-trigger so you really only need to consider the individual triggers you enable